systemtest/src/test/java/io/strimzi/systemtest/ConnectClusterIT.java

```diff
@@ -84,6 +87,24 @@ public void testDeployConnectClusterViaTemplate() {
                 "KAFKA_CONNECT_BOOTSTRAP_SERVERS", KAFKA_CONNECT_BOOTSTRAP_SERVERS));
         String deploymentName = clusterName + "-connect";
         oc.waitForDeployment(deploymentName);
+
+        //Verifying docker image for cluster-operator
+        JsonNode deploymentYaml = yamlFileToJSON(DEPLOYMENT_CONFIG);
+        String coImgNameFromYaml = deploymentYaml.findValue("image").toString();
+        String coImgNameFromPod = getImageNameFromPod(kubeClient.listResourcesByLabel("pod",
+                "name=strimzi-cluster-operator").get(0)).toString().replaceAll("[\\[\\]\\\\]", "");
```

Comment:
You can skip `.toSring()` in this line because method getImageNameFromPod() already returns `String`.